recipes/arraylake/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "arraylake" %}
+{% set version = "0.7.2a1" %}
```

Comment:
Falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
@dcherian are you planning on a stable release? Or do you want this under a dev label?